timeseries/blueflood/blueflood.go

```diff
@@ -66,6 +54,16 @@ type Config struct {
 	MaxSimultaneousRequests int `yaml:"simultaneous_requests"`
 }
 
+type BluefloodParallelRequest struct {
+	limit   int
+	tickets chan struct{}
+}
+
+type httpClient interface {
```

Comment:
😢 Really wish Go would wrap the stdlib in interfaces to allow for stuff like this instead of having to arbitrarily define types.